README.md

```diff
@@ -1,3 +1,7 @@
+# Multicore GIF Encoder
+
+Faster version of the original GIF encoder for usage in production renders.
+
```

Comment:
I'd like to avoid introducing a new header here, perhaps we could mention it in the project description below instead?

Reply:
I originally intended to just use this fork for my own purposes, this is an artifact that needs to be removed. We should point out that it uses multicore CPU for encoding, but not decoding at the moment.

Reply:
Perhaps a future commit will let users suppress usage of multiple cores, e.g. for power efficiency or reducing latency in some extremely exotic situation with 2x2 pixel images.